pkg/kubectl/cmd/cmd.go

```diff
@@ -598,6 +602,54 @@ func NewKubectlCommand(in io.Reader, out, err io.Writer) *cobra.Command {
 		}
 	}
 
+	// TODO(knight42): handler error properly
```

Comment:
The error should be easy to handle.  An error at this point implies an error in the code, probably because more than one completion was registered for the same flag.  You can actually do a `log.Fatal` or something like that.  Since this will happen as soon as kubectl starts (I think), the problem will be found right away in development.

That is how helm does it: it just does a `log.Fatal` if this call returns an error.